ArmoryQt.py

```diff
@@ -1823,14 +1817,10 @@ def loadSettings(self):
 
    #############################################################################
    def loadWallets(self):
-      try:
-         # Use centralized function from WalletUtils instead of direct bridge calls
-         loadWalletsForMainApp(self.wallets)
-         self.setupBlockchainService_step1()
-         TheSignalExecution.executeMethod(self.finalizeLoadWallets)
-      except Exception as e:
-         LOGERROR(f"Failed to load wallets: {e}")
-         # Handle error appropriately - could show error dialog or retry
+      if self.wallets.empty():
```

Comment:
This is circular because of the inane inexistant edge case handling in `__init__`. What if the object was created an effectively empty WalletMap cause there are no wallets to load? Will it loop here in perpetuity now, cause `self.wallets.empty()` will always be true

Reply:
There shouldn't even be a `loadWallets` method left, the object shuold be fed a ready to use WalletMap instance at ctor